src/current/v23.2/call.md

```diff
@@ -0,0 +1,58 @@
+---
+title: CALL
+summary: The CALL statement calls a stored procedure, executing the statements in the procedure body.
+toc: true
+keywords:
+docs_area: reference.sql
+---
+
+The `CALL` [statement]({% link {{ page.version.version }}/sql-statements.md %}) invokes a [stored procedure]({% link {{ page.version.version }}/stored-procedures.md %}).
+
+{{site.data.alerts.callout_info}}
+You cannot call a stored procedure from another stored procedure.
```

Comment:
To be clear, stored procs / udfs in general can call other stored procs / udfs, we just haven't implemented it yet.